src/javascripts/ng-admin/Main/config/translate.js

```diff
@@ -38,6 +38,7 @@ export default function translate($translateProvider) {
       'NEXT': 'Next »',
       'DETAIL': 'Detail',
       'STATE_CHANGE_ERROR': 'State change error: {{ message }}',
+      'STATE_FORBIDDEN_ERROR': 'A server 403 error occured: {{ message }}',
```

Comment:
Well 403 error could be something else than permission, so not sure. I will let you debate with @jpetitcolas as this was is wording for the error when we started discussing this issue (https://github.com/marmelab/ng-admin/issues/1031#issuecomment-251101607)